plugins/win-mf/mf-h264.cpp

```diff
@@ -250,20 +250,20 @@ static void MFH264_GetDefaults(obs_data_t *settings)
 #define PROP_DEF(x, y, z) obs_data_set_default_ ## x(settings, y, z)
 	PROP_DEF(int,    MFP_BITRATE,         2500);
 	PROP_DEF(bool,   MFP_USE_LOWLAT,      true);
-	PROP_DEF(int,    MFP_B_FRAMES,        2);
+	PROP_DEF(int,    MFP_B_FRAMES,        0);
```

Comment:
yes yes, I know exactly what these values mean and where they come from

Reply:
Just trust 😂